packages/frontend/src/system-integration/notifications.ts

```diff
@@ -202,8 +284,13 @@ async function flushNotifications(accountId: number) {
   if (notifications.length > notificationLimit) {
     showGroupedNotification(accountId, notifications)
   } else {
-    for (const { chatId, messageId } of notifications) {
-      await showNotification(accountId, chatId, messageId)
+    for (const { chatId, messageId, isWebxdcInfo } of notifications) {
+      await showNotification(
+        accountId,
+        chatId,
+        messageId,
+        isWebxdcInfo ?? false
+      )
     }
```

Comment:
```suggestion
    for (const {
      chatId,
      messageId,
      isWebxdcInfo,
      eventText,
    } of notifications) {
      await showNotification(
        accountId,
        chatId,
        messageId,
        isWebxdcInfo ?? false,
        eventText
      )
    }
```

Otherwise it seems like. `eventText` is unused.

Reply:
No it s required. I oversaw it, since it is included in the getMessageNotificationInfo if the event has a related WebxdcInfoMessage - I had no use case where an event is sent, but the related message is the webxdc message itself, but it seems that is possible and should be supported.